Packages/com.chisel.editor/Chisel/Editor/Components/Base/Editor/ChiselEditorResources.cs

```diff
@@ -0,0 +1,196 @@
+﻿using UnityEngine;
```

Comment:
this file is a helper to load icons for you, making sure it's the right size for the right screensize, right skin etc.